File: LinkedList.java
import java.util.*;
class Node{
    int data;
    Node next;
    Node(int value){
        data = value;
        next = null;
    }
}
public class LinkedList{
    public static void main(String[] args){
    Scanner sc = new Scanner(System.in);
    int n, value;
    n = sc.nextInt();
    Node Head = null;
    Node Tail = null;
    for(int i=0;i<n;i++){
        System.out.print("Enter data no:"+(i+1)+": ");
        value = sc.nextInt();
        Node newNode = new Node(value);
        if(Head == null){
            Head = newNode;
            Tail = newNode;
        }else{
            Tail.next = newNode;
            Tail = newNode;
        }
    }
    Node Temp = Head;
    while(Temp != null){
        System.out.print(Temp.data+" ");
        Temp=Temp.next;
    }
}
}

File: LinkedList.cpp
#include<iostream>
using namespace std;
class Node{
  public:
  int data;
  Node* next;
  Node(int value){
   data = value;
    next = NULL;
  }
};
int main(){
  int n,value;
  Node* Head = NULL;
  Node* Tail = NULL;
  cin>> n;
  for (int i=0;i<n;i++){
    cout<<"enter data no. "<<(i+1)<<": ";
    cin>>value;
    
  Node* newNode = new Node(value);
  
  if(Head == NULL){
    Head = newNode;
    Tail = newNode;
  }else{
    Tail->next = newNode;
    Tail = newNode;
  }
  }
  Node* Temp = Head;
  while(Temp!=NULL){
    cout<<Temp->data<<" ";
    Temp = Temp -> next;
  }
    
  
  return 0;
}